guzzlehttp/psr7/GHSA-q7rv-6hp3-vh96.yaml

```
@@ -0,0 +1,12 @@
+title:     Inproper parsing of HTTP headers
+link:      https://github.com/guzzle/psr7/releases/tag/2.1.1
```

Can you link to the advisory instead as it has more info?

```suggestion
link:      https://github.com/guzzle/psr7/security/advisories/GHSA-q7rv-6hp3-vh96
```